### PR TITLE
Database extensions for manipulating multiple entities

### LINKED ISSUE #50: Add DB extension methods for entities collection
KORM contains database extension methods `AddAsync`, `EditAsync` and `DeleteAsync` that simplify process of adding, editing and deleting of entity by encapsulating and hiding all necessary codebehind (such as manipulating IDbSet).

However, at this time, these methods accept **only single entity**. It would be nice to have overloads of these methods that would accept **IEnumerable of entities**. And also extension methods that would encapsulate bulk insert and bulk update in similar manner.

### PR DESCRIPTION
**Pull request by kubinko**
Added support for insert/update/delete of IEnumerable of entities and bulk insert/update of IEnumerable of entities to `IDatabaseExtensions`.

Closes #50.

### CODE REVIEW

File: src/IDatabaseExtensions.cs [hunk 1]
```diff
@@ -20,6 +21,26 @@ public static class IDatabaseExtensions
...
+        public static async Task AddAsync<TEntity>(this IDatabase database, IEnumerable<TEntity> entities) where TEntity : class
+            => await CommitChangesAsync(database, (IDbSet<TEntity> dbSet) => dbSet.Add(entities));
+
+        /// <summary>
+        /// Adds <paramref name="entities"/> to the database via bulk insert.
+        /// </summary>
+        /// <typeparam name="TEntity">Entities type.</typeparam>
+        /// <param name="database"><see cref="IDatabase"/> instance.</param>
+        /// <param name="entities">Entities to add.</param>
+        public static async Task AddBulkAsync<TEntity>(
```
Comment (satano) [inline src/IDatabaseExtensions.cs]: Change name to `BulkAddAsync`.

File: src/IDatabaseExtensions.cs [hunk 2]
```diff
@@ -71,6 +103,26 @@ public static class IDatabaseExtensions
...
+        public static async Task EditAsync<TEntity>(this IDatabase database, IEnumerable<TEntity> entities) where TEntity : class
+            => await CommitChangesAsync(database, (IDbSet<TEntity> dbSet) => dbSet.Edit(entities));
+
+        /// <summary>
+        /// Edits <paramref name="entities"/> in the database via bulk update.
+        /// </summary>
+        /// <typeparam name="TEntity">Entities type.</typeparam>
+        /// <param name="database"><see cref="IDatabase"/> instance.</param>
+        /// <param name="entities">Entities to edit.</param>
+        public static async Task EditBulkAsync<TEntity>(
```
Comment (satano) [inline src/IDatabaseExtensions.cs]: Change name to `BulkEditAsync`.

File: src/IDatabaseExtensions.cs (hunk 1 above)
Comment (kubinko) [reply src/IDatabaseExtensions.cs]: Done.

File: src/IDatabaseExtensions.cs (hunk 2 above)
Comment (kubinko) [reply src/IDatabaseExtensions.cs]: Done.

**Comment by kubinko** [comment]
> Add info to the [Simplify Adding, Deleting and Editing records](https://github.com/Kros-sk/Kros.KORM#simplify-adding-deleting-and-editing-records) section in readme file. Just general info.

Done.

**Pull request merged**